src/main/scala/scoverage/ScoverageSbtPlugin.scala

```diff
@@ -155,15 +155,9 @@ object ScoverageSbtPlugin extends AutoPlugin {
           // rangepos is broken in some releases of scala so option to turn it off
           if (coverageHighlighting.value) Some("-Yrangepos") else None
         ).flatten
-      } else if (
-        // TODO this is very temporary until support for this gets merged in.
-        // For now we restrict this to this exact SNAPSHOT version which needs
-        // to be published localled in order to test
-        coverageEnabled.value && scalaVersion.value == "3.1.2-RC1-bin-SNAPSHOT"
-      ) {
+      } else if (coverageEnabled.value) {
```

Comment:
Do you think instead of just checking to see if it's enabled here we should also check that it's not an older version of dotc? If not, they might end up thinking this will work and it will set `-coverage-out` but not do anything.

Reply:
Nice everything looks 🟢 ! I think if we can change this up to be more precise, we can get this merged in.